mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/css/main.css

```diff
@@ -282,16 +418,39 @@
 				font-size: 14px;
 			}
 		}
+
+		.remaining-tasks-container {
+			.remaining-tasks-count {
+				border-radius: 0;
+				font-weight: bold;
+				margin: 0;
+				transition: none;
+
+				&:hover {
+					font-size: 14px;
+				}
+			}
+		}
+
+		.task-history {
+			table {
+				box-shadow: 10px 10px 5px #888;
+			}
+		}
 	}
 
-	/* Max width 406px
+	/* Max width 468px
```

Comment:
Please reduce reduce responsive modes to two that are standard in Liferay, @include respond-to(tablet) and @include respond-to(phone), as previously requested in https://github.com/markocikos/liferay-internship/pull/127#issuecomment-302104859